protocol/v2/ssv/validator/non_committee_validator.go

```diff
@@ -74,6 +76,13 @@ func NewCommitteeObserver(msgID spectypes.MessageID, opts CommitteeObserverOptio
 	}
 
 	co.postConsensusContainer = make(map[phase0.Slot]map[phase0.ValidatorIndex]*ssv.PartialSigContainer, co.postConsensusContainerCapacity())
+	co.beaconVoteHashRoots = ttlcache.New(ttlcache.WithTTL[BeaconVoteHashKey, struct{}](time.Minute * 6))
+
+	go func() {
+		for range time.Tick(time.Minute * 1) {
+			co.beaconVoteHashRoots.DeleteExpired()
+		}
+	}()
```

Comment:
can we call `Start()` instead? similarly to other places where we use `ttlcache.Cache`

```suggestion
	go co.beaconVoteHashRoots.Start()
```